### PR TITLE
Update gazebo article

### LINKED ISSUE #251: Update" Using ROS/Gazebo Simulator with SITL"
update [Using ROS/Gazebo Simulator with SITL](http://dev.ardupilot.com/wiki/using-rosgazebo-simulator-with-sitl/) with new information from the team who created [New ArduPilot plugin for Gazebo](http://diydrones.com/profiles/blogs/705844:BlogPost:2151758?id=705844%3ABlogPost%3A2151758&page=-1)

Contact Aurelien ROY and co with reminder if this does not happen automatically in the new year. They will also need logins sorted out.

### LINKED ISSUE #439: Package "gflags_catkin" is required in page "Using ROS/Gazebo Simulator with SITL"
In page "http://ardupilot.org/dev/docs/using-rosgazebo-simulator-with-sitl.html", error "glog_catkin: Cannot locate rosdep definition for [gflags_catkin]" shows while runing "rosdep install --from-paths src --ignore-src --rosdistro indigo -y".

Pull request had created to fix this issue: https://github.com/ArduPilot/ardupilot_wiki/pull/438

Error message:
$ rosdep install --from-paths src --ignore-src --rosdistro indigo -y
ERROR: the following packages/stacks could not have their rosdep keys resolved
to system dependencies:
glog_catkin: Cannot locate rosdep definition for [gflags_catkin]

### PR DESCRIPTION
**Pull request by squilter**
As discussed with @rmackay9, I have updated the article on using Gazebo for Simulation.

I couldn't figure out how to `make` the article on my system (is there a wiki article for that? :smiley: ), so there might be some syntax errors.

Closes #251 
Closes #438 
Closes #439

### CODE REVIEW

File: dev/source/docs/using-gazebo-simulator-with-sitl.rst [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+-------------------------------
+
+We will also need to get a gazebo model of a quadcopter.
+
+::
+
+    cd ~
+    hg clone https://bitbucket.org/osrf/gazebo_models
+    cd gazebo_models
+    checkout aero_testing_john
```
Comment (khancyr) [inline dev/source/docs/using-gazebo-simulator-with-sitl.rst]: miss hg for checkout

File: dev/source/docs/using-gazebo-simulator-with-sitl.rst [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+Start the Simulator
+===================
+
+In one terminal, enter the ArduCopter directory and start the SITL simulation:
+
+::
+
+    cd ~/ardupilot/ArduCopter
+    sim_vehicle.sh -f gazebo
```
Comment (khancyr) [inline dev/source/docs/using-gazebo-simulator-with-sitl.rst]: use sim_vehicule.py , .sh is going to be removed

**Comment by khancyr** [comment]
slow but working ! 
I extract pluggin from gazebo repo and it seems to works on lastest gazebo and ardupilot master. Hope they merge it soon

File: dev/source/docs/using-gazebo-simulator-with-sitl.rst (hunk 2 above)
Comment (squilter) [inline dev/source/docs/using-gazebo-simulator-with-sitl.rst]: Did you test this with sim_vehicle.py?

Comment (khancyr) [inline dev/source/docs/using-gazebo-simulator-with-sitl.rst]: yes ! And it works.

Comment (squilter) [inline dev/source/docs/using-gazebo-simulator-with-sitl.rst]: Great, I'll update that

**Comment by squilter** [comment]
@khancyr Okay, I made the changes.

@rmackay9

File: dev/source/docs/using-gazebo-simulator-with-sitl.rst [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    This step will not be necessary once this pull request gets merged:
+    <https://bitbucket.org/osrf/gazebo_models/pull-requests/177/aerodynamics-models/diff>`__
+
+
+Set up PATH to build tools
+--------------------------
+
+If you have not already done so, you need to set up the PATH to the build tools
+(located in **/ardupilot/Tools/autotest**) so that the build system can find
+**sim_vehicle.sh**.
```
Comment (khancyr) [inline dev/source/docs/using-gazebo-simulator-with-sitl.rst]: * .py

Comment (squilter) [inline dev/source/docs/using-gazebo-simulator-with-sitl.rst]: I think you're looking at the wrong commit. This was fixed.

Comment (khancyr) [inline dev/source/docs/using-gazebo-simulator-with-sitl.rst]: yep sorry!

**Comment by khancyr** [comment]
hi, which version off gazebo is installed ? 

I found this : https://bitbucket.org/osrf/gazebo_tutorials/pull-requests/360/gazebo_source_install/diff (same author that gazebo APM pluggin)

**Comment by magicrub** [comment]
@squilter this has merge conflicts. Can you update please?

**Comment by khancyr** [comment]
@squilter @magicrub I recommend to wait a little more befor merging since those patch are already outdated... let's OSRF guys freeze their change before merging this ! 
Actually, it needs gazebo ardupilot branch
and gazeb-models master (or zephyr-demos but it isn't working for copter now..)

**Comment by magicrub** [comment]
should https://github.com/ArduPilot/ardupilot_wiki/pull/438 be closed?

**Comment by squilter** [comment]
closing in favor of #572

**Comment by ELSHAIKHI** [comment]
Hello ,
I am trying to download the Additional Baxter packages for MoveIt, written entirely in C++ (pick and place tutorial) , I stuck with this error could you please update me how to fix it ? 
 
ERROR: the following packages/stacks could not have their rosdep keys resolved
to system dependencies:
baxter_moveit_scripts: Cannot locate rosdep definition for [baxter_control]
baxter_pick_place: Cannot locate rosdep definition for [baxter_control]

Regards
Thanks in advanced

**Pull request closed without merging**